src/Engine/ProtoCore/Namespace/ElementRewriter.cs

```diff
@@ -79,6 +79,10 @@ node.datatype.UID < (int)PrimitiveType.kMaxPrimitives)
 
         public override AssociativeNode VisitIdentifierListNode(IdentifierListNode node)
         {
+            IdentifierListNode newIdentifierListNode = null;
```

Comment:
@junmendoza yes exactly. This call is a first pass attempt to resolve the incoming `identifierListNode` so that if it resolves as is, there is no need to traverse the AST deeper.